app/models/validations/financial_validations.rb

```diff
@@ -58,4 +58,19 @@ def validate_tshortfall(record)
       record.errors.add :tshortfall, I18n.t("validations.financial.hbrentshortfall.outstanding_no_benefits")
     end
   end
+
+  def validate_rent_amount(record)
+    if record.brent.present? && record.tshortfall.present? && record.brent < record.tshortfall * 2
+      record.errors.add :brent, I18n.t("validations.financial.rent.less_than_double_shortfall", tshortfall: record.tshortfall * 2)
+      record.errors.add :tshortfall, I18n.t("validations.financial.tshortfall.more_than_rent")
+    end
+
+    if record.scharge.present? && record.this_landlord? && record.weekly_value(record.scharge).present?
+      if !record.weekly_value(record.scharge).between?(0, 55) && record.is_general_needs?
```

Comment:
Do these numbers mean anything? Could they be named constants?

Reply:
Hmmm, will deffo refactor it into something that pulls these from a hash or something, cause there are a few more of these. Not sure what they are other than just ranges